src/main/java/strategy/operationhandlers/OperationHandler.java

```diff
@@ -0,0 +1,7 @@
+package strategy.operationhandlers;
+
+import model.FruitTransaction;
+
+public interface OperationHandler {
+    void getOperationQuantity(FruitTransaction transaction);
```

Comment:
If your method name starts with "get" it can't be void. Rename it. 
Interface called OperationHandler. So, we are going to handle operation may be. But we have one method and we do not pass Operation to it. We pass Transaction. All the names must be consistent and they should show us what method or interface do and what they use for work.